.github/workflows/publish.yml

```diff
@@ -32,11 +32,69 @@ jobs:
         run: echo "TAG=$(echo ${GITHUB_REF:10})" >> $GITHUB_ENV
 
       - name: Publishing app to Google Play
+        env:
+          KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
+          KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
+          KEY_STORE_PASSWORD: ${{ secrets.KEY_STORE_PASSWORD }}
+          DWDS_HTTP_BASIC_ACCESS_AUTHENTICATION: ${{ secrets.DWDS_HTTP_BASIC_ACCESS_AUTHENTICATION }}
+        run: |
+            cd kiwix-android
+            eval "./gradlew publish${TAG^}ReleaseBundleWithPlayAssetDelivery"
+
+  publish_dummy_apk:
```

Comment:
Misleading name, we are uploading bundle.

Reply:
Done.